### PR TITLE
Added SteamID Support.

### PR DESCRIPTION
**Pull request by kratos1812**
Now you can limit the acces to a Custom MVP to a single steamid. Example in kento_mvp.cfg. I've seen many people requesting this.

### CODE REVIEW

**Comment by rogeraabbccdd** [comment]
Looks like this PR only support 1 SteamID.
I think multiple SteamID support would be better.

**Comment by kratos1812** [comment]
> Looks like this PR only support 1 SteamID.
> I think multiple SteamID support would be better.

Just pushed a new commit. I tested it and works well(debuged it, etc etc).
I've added a new cvar too `mvp_no_acces_show_mode`.

File: addons/sourcemod/scripting/kento_mvp.sp [hunk 1]
```diff
@@ -276,9 +314,31 @@ void DisplayMVPMenu(int client, int start)
...
-			mvp_menu.AddItem(g_sMVPName[i], g_sMVPName[i]);
+			if(g_hMVPSteamIds[i].Length) // If the MVP has any "restrictions"
+			{
+				if(g_hMVPSteamIds[i].FindString(steamid) != -1)
+				{
+					mvp_menu.AddItem(g_sMVPName[i], g_sMVPName[i]);
+				}
+				else
+				{
+					switch(Cvar_ShowMode.IntValue)
```
Comment (rogeraabbccdd) [inline addons/sourcemod/scripting/kento_mvp.sp]: I think hook cvar change and store value in a variable is better than calling `.IntValue` everytime.

**Review by rogeraabbccdd — CHANGES_REQUESTED** [review]

**Comment by rogeraabbccdd** [comment]
LGTM

**Pull request merged**